### ISSUE TITLE
Confusion about combining titlo half

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [typiconman/fonts-cu#3]
Hello, 

I have encountered a confusion about combining titlo half, because there is a different font behavior for them.

The [Spasskaya tower](https://en.wikipedia.org/wiki/Spasskaya_Tower) have an icon including Cyrillic [nomina sacra](https://en.wikipedia.org/wiki/Nomina_sacra) marked with long titlo:

* ІС ХС (“Jesus Christ”)

In the develop version of GNU FreeFont, the annotation can be implement in this way: `І<U+FE2E, U+FE2F>С Х<U+FE2E, U+FE2F>С`, but it seems not work with your fonts, maybe can be implement in this way:  `І<U+FE2E>С<U+FE2F> Х<U+FE2E>С<U+FE2F>`, but the titlo seems broken. I don’t know which combinations looks proper.

P.S. You can get the develop version of GNU FreeFont from here, Microsoft Windows user can extract the font file by 7-zip: https://code.launchpad.net/~dns/+recipe/freefont

**Comment by KrasnayaPloshchad** [comment]
Now I think `<U+FE2E, U+FE2F>` can be generated via a mark to mark table, it’s suitable for all caps style.

**Comment by typiconman** [comment]
The correct way to encode the Double Titlo is: 
І<U+FE2E>С<U+FE2F> Х<U+FE2E>С<U+FE2F>

**Comment by KrasnayaPloshchad** [comment]
I just using Ponomar Unicode, I don’t know whether Fevodosk Unicode appearing this problem.

**Comment by typiconman** [comment]
Works as expected in Firefox 42.0 on Linux Mint 17 Qiana, other than the poor positioning over the I, which needs to be fixed.
![shot](https://cloud.githubusercontent.com/assets/5428695/11808268/797eda9a-a330-11e5-9e3b-628c50697eab.png)
 Sounds like a problem with your rendering of OpenType features. What browser / OS are you using?

**Comment by KrasnayaPloshchad** [comment]
I have used Firefox 40.0 on Windows 10 when I post this issue. Now I was updated Firefox to 42.0, but I still unhappy with positioning. 
![Screenshot 2.PNG](https://ooo.0o0.ooo/2015/12/15/567046aae7305.png)

**Comment by KrasnayaPloshchad** [comment]
![to my eyes you should adjust the height for them](https://cloud.githubusercontent.com/assets/10364351/11817962/8a072716-a392-11e5-9ce9-3594ceef8932.png)

**Comment by mamyt** [comment]
In Windows 10, it seems to depend on the browser that is used. Microsoft Edge (the latest from Microsoft in terms of browsers) for once actually gives the best result. Firefox is okay, but Chrome fails completely. Examples are posted below.
[TestingCombiningTilde.pdf](https://github.com/typiconman/fonts-cu/files/62967/TestingCombiningTilde.pdf)

**Comment by typiconman** [comment]
Yes, there is definitely some work that can be done on adjusting the heights. But that is actually kind of difficult to do in OpenType.

**Comment by KrasnayaPloshchad** [comment]
![you should also make them happy as you expected in your screenshot](https://cloud.githubusercontent.com/assets/10364351/11871804/dd973864-a50c-11e5-80d3-4e1de495ce4d.png)

**Comment by typiconman** [comment]
You are not entering them correctly. It should be I<U+FE2E>C<U+FE2F>.

**Comment by typiconman** [comment]
Too complicated to fix, so I'm bumping this to the next release.

**Comment by KrasnayaPloshchad** [comment]
You can try to create a branch repo for this work, then other people can help.

**Comment by KrasnayaPloshchad** [comment]
Recently OpenType specification introduced OpenType Variation technology, does it help?

**Comment by typiconman** [comment]
I suppose you could use the Variations to create expanding widths of the titlo glyph.
https://medium.com/@tiro/https-medium-com-tiro-introducing-opentype-variable-fonts-12ba6cd2369#.3pn2hj1m8
However, this is still very low priority on the list of things to support.

**Comment by StefanPeev** [comment]
@typiconman @KrasnayaPloshchad As far as I can see the problem is not solved still. I offered my suggestions in #75. Well, actually the glyphs /uniFE2E and /uniFE26 can connect correctly above two letters without the help of /uniFE2F if /uniFE2E and /uniFE26 are a little bit longer then now and if the anchors are placed correctly. However a little compromise is necessary to get things right for all possible cases. The horizontal bar of the diacritics Titlo (/uniFE2E and /uniFE26) must be strictly horizontal. In this way, we will be sure that, in all possible cases, there will be no discrepancies in one direction or the other at the place of assembly.

**Comment by typiconman** [comment]
@StefanPeev attempts to fix this are welcome. 
As I have played around with this quite a bit previously, I'll warn right away that it is quite difficult to do. Yes you can make the titlo halves longer, but what they are above narrow characters, like the і?
As I said above, I don't see this as a big priority since titlos over two or more characters are not used in Synodal orthography, for which this font is designed. The effort may be better spent fixing titlo positioning in an ustav-era font such as Shafarik or Menaion, or in creating a decorative font for iconographers to use.
But, again, pull requests are always welcome.

**Comment by typiconman** [comment]
See also the discussion in #25 and other threads referred to in this one.

**Comment by StefanPeev** [comment]
@typiconman If we have a good solution for Ponomar Unicode the same method can be used for any other font. The situation can be solved in FontLab 8 very simply: we add an anchor point to the first base letter in such a way that the Titlo diacritic /uniFE2E to be left justified. On the other hand we add an anchor point to the second base letter in such a way that the Titlo diacritic /uniFE26 to be right justified. If the horizontal bars of /uniFE2E and /uniFE26 are long enough, they will stick together in a nice way. I'll see how to do this in FontForge also.
